src/topology/basic.lean

```diff
@@ -426,22 +426,28 @@ end
 ### Neighborhoods
 -/
 
-/-- neighbourhood filter -/
+/-- The neighborhoods of a point `a` are the sets which include an open set containing `a`. These
+sets form a filter. -/
 def nhds (a : α) : filter α := (⨅ s ∈ {s : set α | a ∈ s ∧ is_open s}, 𝓟 s)
 
 localized "notation `𝓝` := nhds" in topological_space
 
 lemma nhds_def (a : α) : 𝓝 a = (⨅ s ∈ {s : set α | a ∈ s ∧ is_open s}, 𝓟 s) := rfl
 
+/-- The open sets containing `a` are a basis for the neighborhood filter. See `nhds_basis_opens'`
+for a variant using open neighborhoods instead. -/
 lemma nhds_basis_opens (a : α) : (𝓝 a).has_basis (λ s : set α, a ∈ s ∧ is_open s) (λ x, x) :=
 has_basis_binfi_principal
   (λ s ⟨has, hs⟩ t ⟨hat, ht⟩, ⟨s ∩ t, ⟨⟨has, hat⟩, is_open_inter hs ht⟩,
     ⟨inter_subset_left _ _, inter_subset_right _ _⟩⟩)
   ⟨univ, ⟨mem_univ a, is_open_univ⟩⟩
 
+/-- A filter has the neighborhoods of `a` iff it has the open sets containing `a`. -/
 lemma le_nhds_iff {f a} : f ≤ 𝓝 a ↔ ∀ s : set α, a ∈ s → is_open s → s ∈ f :=
 by simp [nhds_def]
 
+/-- A filter is included in the neighborhood filter if it is included in one of the principal
+filters that comprise it. -/
```

Comment:
I'm pretty sure this is correct, as `≤` means `⊇` (includes) for filters. But I'm open (haha) to better ways to word this.